test/nodes/test_connector.py

```diff
@@ -36,7 +40,9 @@ def content_match(crawler: Crawler, url: str, crawled_page: Path):
         return page_data["content"] == expected_crawled_content
 
 
-def content_in_results(crawler: Crawler, url: str, results: List[Path], expected_matches_count=1):
+def content_in_results(
+    crawler: Crawler, url: str, results: List[Path], expected_matches_count=1, loading_wait_time: int = None
```

Comment:
I think `loading_wait_time` is unused now

Reply:
Oh, I think this got added after a merge from selenium-upgrade branch. Sorry for not filtering before.